open-metadata-implementation/user-interfaces/ui-admin-services/docs/user/configuring-configuration-file-store.md

```diff
@@ -0,0 +1,91 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Overriding the location for storing configuration documents
+
+By default, [configuration documents](../concepts/configuration-document.md) are stored as a
+file in the OMAG server platform's home directory.
+The file name is 
+
+```
+omag.server.{serverName}.config
```

Comment:
Examples, filenames, ports should be UI centric

Reply:
Do you want to make the change here? If not we can raise a new issue to track?

Reply:
I have a separate issue - to test and confirm whether this works . I have made cosmetic changes as requested

Reply:
#2255 I assume